_posts/2022-06-15-ejb-over-https.adoc

```diff
@@ -0,0 +1,340 @@
+---
+layout: post
+title: "EJBs over HTTPS: Securing Jakarta Enterprise Beans with mutual TLS authentication"
+date: 2022-06-15
+tags: ejb tls sasl https credential-store key-store trust-store
+synopsis: A demonstration on how to configure Elytron for secure connections to EJBs, using mutual TLS, SASL authentication mechanisms, and credential stores.
+author: cam-rod
+---
+
+The *https://github.com/wildfly-security-incubator/elytron-examples/tree/master/ejb-https[ejb-https]* example
+demonstrates how to use the Elytron subsystem to secure connections to EJBs with mutual TLS
```

Comment:
s/how to use the Elytron subsystem to secure connections to EJBs/how to use Elytron to secure EJBs